observability/README.md

```diff
@@ -273,6 +272,13 @@ kubectl apply -f ../distributed-calculator/deploy
 
 <!-- END_STEP -->
 
+Then, open the distributed calculator UI.
+
+If this is the first time trying the distributed calculator, find more detailed instructions in the [distributed-calculator](https://github.com/dapr/quickstarts/blob/master/distributed-calculator/README.md) quickstart.
+
+
+The quick version for starting the calculator quickstart:
```

Comment:
I'm not sure this line makes sense anymore? Probably can just remove it.

Reply:
Yep.